### DISCUSSION TITLE
Separate package identifies for different dotnet SDKs

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [microsoft/winget-pkgs#40771] [Ideas] [answered]
Wouldn't be better for updates/maintenance for devs and it teams? e.g v3.1 to 6.1 all have same identifier.

**Comment by OfficialEsco** [comment]
* https://github.com/microsoft/winget-pkgs/issues/17985

**Reply by kdpuvvadi** [reply]
So, what's the latest on that?

**Reply by OfficialEsco** [reply]
I wish i knew... IMO we're waiting on winget-cli to catch up

**Comment by denelon** [comment, marked as answer]
We are actively working with the .NET team to have them become the "verified developer" for the .NET packages. They will then take over publishing the manifests to ensure the versions and URLs are correct.

**Reply by denelon** [reply]
The .NET team is now maintaining their own packages.

**Reply by kdpuvvadi** [reply]
Does it mean different identifiers for different versions?

**Reply by OfficialEsco** [reply]
Yes. Anything that can be installed side-by-side should be a different package. The Microsoft Team have already splitted .NET https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/DotNet
